docs/openshift_ci.md

```diff
@@ -2,10 +2,49 @@
 
-CI jobs are represented by a GitHub status check on PRs and will report a pending/pass/fail state per job. Jobs are loosely broken up by their maintaining OKD/K8S SIG, which is embedded in the status check’s name. Failed jobs will toggle their respective git status as failed and block the PR from merging. Blocks may still be overridden by maintainers.
+CI jobs are represented by a GitHub status check on PRs and will report a pending/pass/fail state per job. Jobs are currently loosely broken up by their maintaining OKD/K8S SIG, which is embedded in the status check’s name. Failed jobs will toggle their respective git status as failed and block the PR from merging. Blocks may still be overridden by maintainers. Full list of tests can be found [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/microshift/openshift-microshift-main.yaml#L92-L141).
 
 E2e jobs are executed in parallel, each against its own isolated MicroShift cluster. That is, there are as many MicroShift clusters deployed as there are SIG status checks. Expect a complete PR test run to take approximately 45 minutes.
 
+## Running locally
+Prior to opening a PR it is useful and efficient to be able to run tests locally relying on your own resources. In order to do it you need a running instance of MicroShift and a valid version of [openshift-tests](https://github.com/openshift/origin).
+> `openshift-tests` follows the same branching strategy than the rest of the product, so be sure to check base version in `pkg/release/release.go`.
+
+`openshift-tests` requires a kubeconfig and `oc` binary as well. MicroShift generates a kubeconfig, located in `/var/lib/microshift/resources/kubeadmin/kubeconfig`, however this is only valid for local access, as it holds these valid names in a RHEL4Edge deployment:
+```
+Subject: O = 127.0.0.1 + O = localhost + O = 192.168.122.108 + O = localhost.localdomain, CN = kubelet
+...
+X509v3 Subject Alternative Name:
+    DNS:localhost, DNS:localhost.localdomain, DNS:192.168.122.108, DNS:127.0.0.1, IP Address:192.168.122.108, IP Address:127.0.0.1
+```
+Which means we must setup the kubeconfig to use one of those. If you are running `openshift-tests` from within the same machine where `MicroShift` is running, then no extra tasks are needed. If this is not the case then you need to either configure new entries in `/etc/hosts`, or use ssh tunnels.
+
+For example, imagine we have a `microshift-edge` machine where we are running `MicroShift` and we want to run `openshift-tests` from a development machine. If we were to use ssh tunnels we would run:
```

Comment:
I think we should separate these two cases. Let's start with the "simple" case where we run locally and explain how to do this. 
The more complex remote case should be a separate sub-chapter and explain what needs to be done to make it work.

Reply:
Makes sense, will separate them.